core/src/main/java/org/apache/iceberg/rest/responses/FetchScanTasksResponse.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.rest.responses;
+
+import java.util.List;
+import org.apache.iceberg.DeleteFile;
+import org.apache.iceberg.FileScanTask;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+import org.apache.iceberg.rest.RESTResponse;
+
+public class FetchScanTasksResponse implements RESTResponse {
+
+  private List<String> planTasks;
+
+  private List<FileScanTask> fileScanTasks;
+
+  private List<DeleteFile> deleteFiles;
+
+  public FetchScanTasksResponse() {
+    // Needed for Jackson Deserialization.
+  }
+
+  public FetchScanTasksResponse(
```

Comment:
Since these are following a builder pattern, I think these constructors should be private.